ticketing-eventing/handler-core/src/main/java/de/muenchen/oss/dbs/ticketing/eventing/handlercore/application/port/out/TicketingOutPort.java

```diff
@@ -1,13 +1,15 @@
 package de.muenchen.oss.dbs.ticketing.eventing.handlercore.application.port.out;
 
 import de.muenchen.oss.dbs.ticketing.eai.client.model.TicketInternal;
-import de.muenchen.oss.dbs.ticketing.eai.client.model.UpdateTicketDTO;
+import de.muenchen.oss.dbs.ticketing.eai.client.model.UpdateTicketDTOV2;
 import java.io.InputStream;
+import java.util.Collection;
+import org.springframework.core.io.AbstractResource;
 
 public interface TicketingOutPort {
     TicketInternal getTicket(String ticketId);
 
-    TicketInternal updateTicket(UpdateTicketDTO updateTicketDTO);
+    TicketInternal updateTicket(UpdateTicketDTOV2 updateTicketDTO, Collection<AbstractResource> attachments);
```

Comment:
@rowe42 I don't really know if it's necessary but we should propably be thing about if we should add some adapter classes to not be directly dependent on the one from the API.
Do have any opinion on that? Currently this change requires also a change in the mail-handler-service